### ISSUE TITLE
Show TOC in sidebar

### ISSUE THREAD

**Issue by mluettecke** [LotarProject/dokuwiki-template-bootstrap3#41]
This is more a feature request.

Would it be possible to show the table of contents in the sidebar? I made it possible with an ugly hack. It would be prettier to add it the options. I added a screenshot of how it looks right now.

![tocinsidebar](https://cloud.githubusercontent.com/assets/4482510/8217390/838e6086-153d-11e5-9ebe-b6b631113f78.PNG)

**Comment by LotarProject** [comment]
Hi! I have tested different solutions but the easy solution is to use the Inline TOC plugin. Few days ago i have added the support for this plugin (see #115).

You can add in sidebar page the ``{{INLINETOC}}`` tag. In addition this plugin automatically hide the main TOC.

I hope this solution is the most helpful.

Best regards,
Joseph

**Comment by Frozenlock** [comment]
Isn't `{{INLINETOC}}` for the current page (in this case `sidebar)?

I added the tag into my `sidebar` page, but I don't see the TOC for the main content.

**Comment by raychani** [comment]
I've tested [inlineTOC plugin](https://www.dokuwiki.org/plugin:inlinetoc) with this template.
But the plugin and this template don't work fine.
I added the tag into my sidebar. I can see the TOC.
But sometimes the space(that displayed toc) was remained.

**Comment by igittigitt** [comment]
I also failed to get "Inline TOC" Plugin running with bootstrap3/cerulean. I put {{INLINETOC}} in rightsidebar. After editing and saving a page, the TOC is displayed one time in the sidebar and not in the page. Successive page loads show the TOC on page again and not in sidebar.

Another thing is the layout of the TOC when rendered by the plugin. I set my TOC to display only H2-levels. With three H2 headlines in the page, the lines in TOC following the first line, are intendet.
